src/StdJson.sol

```diff
@@ -68,4 +68,80 @@ library stdJson {
     function readBytesArray(string memory json, string memory key) internal pure returns (bytes[] memory) {
         return abi.decode(vm.parseJson(json, key), (bytes[]));
     }
+    function serialize(string memory jsonKey, string memory key, bool value) internal returns (string memory) {
```

Comment:
Nit on adding a blank line
```suggestion

    function serialize(string memory jsonKey, string memory key, bool value) internal returns (string memory) {
```

So the TLDR usage of this library, with `using stdJson for string;` is:
- To read JSON: Use `parseRaw` and the `read*` methods
- To write JSON: Use `write` and the `serialize` methods

Is that correct? If so let's add some comments at the top of the file with that info so its easier for devs to parse/skim this file